media-sound/wav2json/metadata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>sound@gentoo.org</email>
```

Comment:
Please do not add Gentoo projects or developers as maintainers of your packages if you haven't talked to them first. Instead, add yourself and the proxy-maint project.